.travis/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+( yarn prettify:check || yarn lint || true ) && yarn test
+
+if [ "$TRAVIS_BRANCH" = "master" ];
+then
+    
+    git config --global user.email "travis@travis-ci.org"
+    git config --global user.name "Travis CI"
+    
+    yarn prod
```

Comment:
Is it possible to DRY ` yarn prod`? Why `yarn prod` can't be launched before `if` statement?
It is duplicated in the `else` section.